types/schemas/prototypeApplication/data/Applicant.ts

```diff
@@ -40,7 +40,12 @@ export type LDCApplicant = ApplicantBase & {
    * @description Information about the property owners, if different than the applicant
    */
   ownership:
-    | {interest: Extract<OwnersInterest, 'owner'>}
+    | {interest: Extract<OwnersInterest, 'owner'>} // sole owner does not report `owners`
+    | {
+        interest: Extract<OwnersInterest, 'other'>;
+        interestDescription: string; // "other" interest uniquely requires a free text description
+        owners: (OwnersNoticeGiven | OwnersNoNoticeGiven)[];
+      }
```

Comment:
The `prototypeApplication` is not actually exchanged with BOPS yet, but this is a nice example of how we're able to accomplish stronger types with it - we can specifically specify that when the interest is "other" within an LDC then there will be a freetext requirement "interestDescription".

Whereas for the current `application` schema below we simply introduce `interestDescription` as always optional (since PP "other" interest does _not_ require the freetext!)